t/run/switch-I-and-M.t

```diff
@@ -0,0 +1,21 @@
+#!./perl
+
+BEGIN {
+    chdir 't' if -d 't';
+    @INC = '../lib';
+}
+
+use strict;
+
+require './test.pl';
+
+plan(1);
+
+$ENV{PERL5OPT} = "-Mlib=optm1 -Iopti1 -Mlib=optm2 -Iopti2";
+$ENV{PERL5LIB} = "e1:e2";
+
+# this isn't *quite* identical to what's in perlrun.pod, because
+# test.pl:_create_runperl adds -I../lib and -I.
+like(runperl(switches => [qw(-Ii1 -Mlib=m1 -Ii2 -Mlib=m2)], prog => 'print join(chr(32), @INC)'),
```

Comment:
The weird `chr(32)` is here because `" "` isn't allowed - quotes in `prog`, which ends up being quoted as a `-e` argument, are not portable.

Reply:
Could that not be written more clearly as `q( )`?

Reply:
You're right, it could! And I just realised that `32` is an ASCII-ism and we still support EBCDIC.